staging/src/k8s.io/apiserver/pkg/apis/cel/config.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cel
+
+const (
+	// PerCallLimit specify the actual cost limit per CEL validation call
+	// current PerCallLimit gives roughly 0.1 second for each expression validation call
+	PerCallLimit = 1000000
+
+	// RuntimeCELCostBudget is the overall cost budget for runtime CEL validation cost per ValidatingAdmissionPolicy or CustomResource
+	// current RuntimeCELCostBudget gives roughly 1 seconds for the validation
+	RuntimeCELCostBudget = 10000000
```

Comment:
Can we include a unit into these constant names? Even if it's `PerCallLimitFooBars` or `PerCallLimitApproximateSeconds`.

Reply:
It is a little tricky to define the unit.. The cost is the cost of operation defined by cel-go. Open to suggestions on this one :)

Reply:
I don't know if this helps, but I'll make a few statements about cost:

- CEL cost units represent an evaluation of a basic CEL compute operation. CEL cost units do not have a 1:1 correspondence with machine instructions (or CPU cycles), but each can be though of roughly as the compute required for CEL to evaluate simple operations like a integer comparison.
- For any CEL expression and input, the CEL cost will always be exactly the same regardless or platform or system load.